spec/controllers/api/current/appointments_controller_spec.rb

```diff
@@ -34,6 +34,36 @@ def create_record_list(n, options = {})
   describe 'POST sync: send data from device to server;' do
     it_behaves_like 'a working sync controller creating records'
     it_behaves_like 'a working sync controller updating records'
+
+    context 'handles appointment_type correctly' do
+      before :each do
+        set_authentication_headers
+      end
+
+      describe 'stores appointment_type correctly' do
+        let(:request_key) { model.to_s.underscore.pluralize }
+        let(:new_records) { (1..5).map { build_payload.call } }
```

Comment:
`5.times { ... }`

Reply:
Done.

Reply:
Actually reverted this since we are storing the created records for use in the sync payload. 5.times will return 5 as the value, so map is actually correct here.